cmd/av/pr_queue.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/aviator-co/av/internal/utils/colors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/aviator-co/av/internal/actions"
+	"github.com/aviator-co/av/internal/avgql"
+	"github.com/shurcooL/graphql"
+	"github.com/spf13/cobra"
+)
+
+var prQueueFlags struct {
+	SkipLine bool
+	Targets  string
+}
+
+var prQueueCmd = &cobra.Command{
+	Use:          "queue",
+	Short:        "queue a pull request for the current branch",
+	SilenceUsage: true,
+	Args:         cobra.NoArgs,
+	// error or reterr from emperror.dev/errors here?
+	RunE: func(cmd *cobra.Command, args []string) error {
+		repo, err := getRepo()
+		if err != nil {
+			return err
+		}
+
+		db, err := getDB(repo)
+		if err != nil {
+			return err
+		}
+
+		tx := db.ReadTx()
+		currentBranchName, err := repo.CurrentBranchName()
+		if err != nil {
+			return err
+		}
+
+		branch, _ := tx.Branch(currentBranchName)
+		if branch.PullRequest == nil {
+			return errors.New(
+				"this branch has no associated pull request (run `av pr create` to create one)",
+			)
+		}
+
+		prNumber := branch.PullRequest.Number
+		repository, exists := tx.Repository()
+		if !exists {
+			return actions.ErrRepoNotInitialized
+		}
+
+		var variables = map[string]interface{}{
+			"repoOwner": graphql.String(repository.Owner),
+			"repoName":  graphql.String(repository.Name),
+			"prNumber":  graphql.Int(prNumber),
+		}
+
+		// I have a feeling this would be better written inside of av/internals
```

Comment:
Remove this comment~ we can move things to `internal` at some point if we want to.